js/cedar.js

```diff
@@ -303,23 +312,23 @@ Cedar.Store.prototype.refresh = function() {
 // Get content objects from server and save to local storage
 Cedar.Store.prototype.getRemote = function(options) {
   var defaultOptions = {
-    path: '/queries/contententries/'
+    path: '/queries/all/'
   };
   options = $.extend({}, defaultOptions, options);
   var defaultParams = {};
   var params = $.extend({}, defaultParams, {
-    guidfilter: options.filter
+    filter: "Mazlo App :",
```

Comment:
definately don't want to have any mazlo specific stuff in here.  do we need to think about creating a mazlo fork of the main cedar js project?